applications/IGAStructuralMechanicsApplication/custom_elements/base_discrete_element.cpp

```diff
@@ -0,0 +1,559 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/IGAStructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Tobias Teschemacher
+//
+
+
+// System includes
+#include "includes/define.h"
+#include "utilities/math_utils.h"
+
+// External includes
+
+// Project includes
+#include "custom_elements/base_discrete_element.h"
+
+#include "iga_structural_mechanics_application.h"
+#include "iga_structural_mechanics_application_variables.h"
+
+#include "geometries/geometry.h"
+
+namespace Kratos
+{
+    void BaseDiscreteElement::Initialize()
+    {
+        KRATOS_TRY
+
+        //Constitutive Law initialisation
+        if (mConstitutiveLawVector.size() != 1)
```

Comment:
I recommend to use brackets also for one-liner

```
} else {
    // ...
}
```

Reply:
we usually don't do that

Reply:
We do in fact @roigcarlo

Reply:
is this documented in the StyleGuide?
Would be nice then we don't have to discuss it again each time...